components/director/internal/domain/oauth20/resolver.go

```diff
@@ -106,15 +112,17 @@ func (r *Resolver) generateClientCredentials(ctx context.Context, objType model.
 	}
 
 	id := clientCreds.ClientID
+	r.logger.Debugf("Creating SystemAuth model for the client credentials for %s with id %s", objType, objID)
```

Comment:
`model` seems a little confusing probably redundant. You just create a SystemAuth (valid for the logs below also)